packages/utils/src/crypto.ts

```diff
@@ -129,3 +129,18 @@ export const genEpochTreeLeaf = (
     }
     return polysum
 }
+
+export const genNullifier = (
```

Comment:
I think we shouldn't add these functions to the utils package. The nullifier can be calculated using just `hash2` and the id commitment can be calculated using `@semaphore-protocol/identity`

Reply:
Okay, that makes sense. Am I supposed to rebase this branch to main? `git rebase main` says `Current branch prevent-double-action is up to date.`

Reply:
Yes, make sure to pull the latest `main` before trying to rebase

Reply:
ohh sorry for such a dumb question. okay i'll rebase